packages/forklift-console-plugin/src/modules/Plans/views/create/PlanCreatePage.tsx

```diff
@@ -20,15 +24,27 @@ import './PlanCreatePage.style.css';
 export const PlanCreatePage: React.FC<{ namespace: string }> = ({ namespace }) => {
   // Get optional initial state context
   const { data } = useCreateVmMigrationData();
-  const history = useHistory();
-  const startAtStep = data?.provider !== undefined ? 2 : 1;
+  const createPlanFromPlansList = data?.provider !== undefined ? false : true;
+  const startAtStep = createPlanFromPlansList ? 1 : 2;
   const [activeNamespace, setActiveNamespace] = useActiveNamespace();
   const defaultNamespace = process?.env?.DEFAULT_NAMESPACE || 'default';
   const projectName =
     data?.projectName ||
     (activeNamespace === '#ALL_NS#' ? 'openshift-mtv' : activeNamespace) ||
     defaultNamespace;
 
+  const plansListURL = getResourceUrl({
+    reference: PlanModelRef,
+    namespace: namespace,
+    namespaced: namespace !== undefined,
+  });
+
+  const providerURL = getResourceUrl({
+    reference: ProviderModelRef,
+    name: data?.provider?.metadata?.name,
+    namespace: data?.provider?.metadata?.namespace,
```

Comment:
can be in a util file

Reply:
The implementation of `getResourceUrl` is already in a [util](https://github.com/kubev2v/forklift-console-plugin/blob/main/packages/forklift-console-plugin/src/modules/Providers/utils/helpers/getResourceUrl.ts) file 
This is only a reference